### PR TITLE
Add linters

### PR DESCRIPTION
**Pull request by Amrit131**
# Section 2 
## In this pull request/branch we have added following features  -:

- Updated index.html file to build a section 2.
- Added styling to the section in style.css file.
- Added a new folder called images.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,5 +16,36 @@
...
+                </div>
+                <div class="socials">
+                    <img src="images/microverse.png" alt="Microverse link" class="social-img">
+                </div>
+                <div id="bg-container">
+                    <img id="bg-img" src="images/Groupbackground.png" alt="Background card">
+                </div>   
+            </div>
+             
+        </section>
```
Comment (Qoosim) [inline index.html]: - [x] It is required to use `ul` to wrap all your social icon, use `li` to list each of them and wrap each of the icons in a link (i.e `a` tag). You don't need to declare a separate `section` for the social icons, they should be inside the section you've already declared.

File: index.html [hunk 2]
```diff
@@ -16,5 +16,36 @@
...
+    </main>
```
Comment (Qoosim) [inline index.html]: - [x] It is required to align the content of this section with `display: flex`. Kindly look into that and fix it.

File: style.css [hunk 3]
```diff
@@ -39,3 +40,71 @@
...
   padding-right: 5px;
 }
+
+#content {
+  width: 375px;
+  height: 590px;
+  background: #1c1a19;
+  margin: 0;
+  position: relative;
+}
```
Comment (Qoosim) [inline style.css]: - [x] It is required to align the content of this section with `display: flex`. Kindly look into that and fix it.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Amrit131, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-   No linter errors. :heavy_check_mark:
-   Good README file. :heavy_check_mark: 
-   Correct GitHub Flow. :heavy_check_mark: 
-   Meaningful commit messages. :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [x] Kindly remove the logo from the `<nav>` tag and wrap your hamburger inside the `<nav>`. You can then give the `header` tag a class name and use that to style them your way. The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices. Kindly take a look :point_down:  You can change it
### from
```
  <header>
      <nav id="top">
            <a href="#home" id="heading">Amrit Mishra</a>
            <div id="button-holder">
                <img src="images/button.png" alt="Hamburger button" id="button">
            </div>
       </nav>
   </header>
```
### to 

```
  <header>
        <a href="#home" id="heading">Amrit Mishra</a>
        <nav id="button-holder">
            <img src="images/button.png" alt="Hamburger button" id="button">
        </nav>
    </header>
```

File: index.html (hunk 1 above)
Comment (Amrit131) [reply index.html]: I have removed the section and moved all my socials to the other section. I have also used ul to wrap all my social icons while using li to list each of them.

File: index.html (hunk 2 above)
Comment (Amrit131) [reply index.html]: I have aligned the content of this section.

File: style.css (hunk 3 above)
Comment (Amrit131) [reply style.css]: @Qoosim I have aligned the content of this section with `display: flex;`.

**Review by ichala — APPROVED** [review]
Hi @Amrit131 ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implement✔️
 - Good Design✔️

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


*************************************
# Optional Suggestions 

- [ ] N/A


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Amrit131** [comment]
@Qoosim @ichala I will have to say getting a review would have been a lot scarier if these GIFs/images were not used by coding reviewers. Thank You!

**Pull request merged**